es/lib/isFQDN.js

```diff
@@ -30,7 +30,7 @@ export default function isFQDN(str, options) {
     } // disallow spaces
 
 
-    if (/[\s\u2002-\u200B\u202F\u205F\u3000\uFEFF\uDB40\uDC20]/.test(tld)) {
+    if (/(?:[\t-\r \xA0\u1680\u2000-\u200B\u2028\u2029\u202F\u205F\u3000\uFEFF]|\uDB40\uDC20)/.test(tld)) {
```

Comment:
This is unrelated too.

Reply:
Reverted back. But bear in mind that this has been generated by running the build script. Eslint is related to this change, if you check the source file under `./src/lib/` you will find no changes.